regolith/schemas.py

```diff
@@ -1167,12 +1167,12 @@
             "type": "string",
         },
         "month": {
-            "description": "what month the person was met",
+            "description": "the month when the entry was connected",
             "required": False,
             "type": "integer",
         },
         "year": {
-            "description": "how year the person was met",
+            "description": "the year when the entry was connected",
```

Comment:
whoops, did I make a typo, it is the "the year when the entry was created".  Sorry about that.

Reply:
Done